modules/augment/src/rotate.cpp

```diff
@@ -0,0 +1,40 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "precomp.hpp"
+namespace cv { namespace augment {
+
+Rotate::Rotate(float minAngle, float maxAngle)
+{
+    this->minAngle = minAngle;
+    this->maxAngle = maxAngle;
+}
```

Comment:
```cpp
Rotate::Rotate(float _minAngle, float _maxAngle) : minAngle(_minAngle), maxAngle(_maxAngle) {}
```

Reply:
I will add _ to the members of the classes instead of the functions arguments to remove the warnings and because the functions parameters are the ones visible to the user

Reply:
This is not a header file. You can use different names between function prototype and body.